qiskit/transpiler/preset_passmanagers/common.py

```diff
@@ -407,6 +408,13 @@ def _direction_condition(property_set):
     return pre_opt
 
 
+def generate_post_op_passmanager():
+    """Generates the post-optimization pass manager."""
+    out = PassManager()
+    out.append(FinalizeLayouts())
+    return out
```

Comment:
Highly unimportant, but just to point out that you can also spell this `return PassManager([FinalizeLayouts()])`.